code/modules/projectiles/magazines/smgs.dm

```diff
@@ -36,6 +36,12 @@
 	icon_state = "m39_le"
 	default_ammo = /datum/ammo/bullet/smg/rubber
 
+/obj/item/ammo_magazine/smg/m39/heap
+	name = "\improper M39 HEAP magazine (10x20mm)"
+	desc = "A 10x20mm caseless armor-piercing high-explosive submachinegun magazine. The bullet tips are made out of a special explosive, designed to penetrate armor then detonate for maximum soft-tissue damage."
```

Comment:
```suggestion
	desc = "A 10x20mm caseless armor-piercing high-explosive submachinegun magazine. The bullet tips are made out of a special explosive material, designed to penetrate armor then detonate for maximum soft-tissue damage."
```
this makes more sense to me but if it's actually dumb just don't change it

Reply:
the tip would realistically have more than just the explosive material, arming mechanism etc, so yeah probably dumb.

Reply:
i can't believe this happened to me